src/olympia/files/migrations/0031_file_status_disabled_reason.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 4.2.7 on 2023-11-07 13:54
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('files', '0030_remove-add-guid-to-manifest-waffle-switch'),
+    ]
+
+    operations = [
+        migrations.AddField(
```

Comment:
To add to push notes: that is going to be an expensive migration, and since it touches `File` we'll need to disconnect the update service while it's running.